### PR TITLE
[CI] Remove the dependency about Debian jessie backport

### PR DESCRIPTION
**Pull request by withsmilo**
* related issue : no.5 in https://github.com/ucbrise/clipper/issues/656

### CODE REVIEW

File: dockerfiles/PySparkContainerDockerfile [hunk 1]
```diff
@@ -6,7 +6,6 @@ FROM ${REGISTRY}/${RPC_VERSION}-rpc:${CODE_VERSION}
 LABEL maintainer="Dan Crankshaw <dscrankshaw@gmail.com>"
 
 RUN mkdir -p /usr/share/man/man1 && \
-    (echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list) && \
     apt-get update -qq -y && \
```
Comment (RehanSD) [inline dockerfiles/PySparkContainerDockerfile]: You don't need this update if you're not adding anything to the apt packages list.
```suggestion
   
```

File: dockerfiles/ClipperPy35DevDockerfile [hunk 2]
```diff
@@ -35,7 +35,6 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s
 
 # Install Java
 RUN mkdir -p /usr/share/man/man1 && \
-    (echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list) && \
     apt-get update -qq -y && \
```
Comment (RehanSD) [inline dockerfiles/ClipperPy35DevDockerfile:38]: Same here.

File: dockerfiles/ClipperDevDockerfile [hunk 3]
```diff
@@ -30,7 +30,6 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s
 
 # Install Java
 RUN mkdir -p /usr/share/man/man1 && \
-    (echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list) && \
     apt-get update -qq -y && \
```
Comment (RehanSD) [inline dockerfiles/ClipperDevDockerfile:33]: Same here.

**Review by RehanSD — CHANGES_REQUESTED** [review]
Please see if the changes that I suggested will work. Once they are made and the CI passes, I can merge this.

File: dockerfiles/PySparkContainerDockerfile (hunk 1 above)
Comment (withsmilo) [reply dockerfiles/PySparkContainerDockerfile]: I missed that the base image(${REGISTRY}/${RPC_VERSION}-rpc:${CODE_VERSION}) has already the updated apt packages list. I will delete the code right now. Thank you, @RehanSD .

File: dockerfiles/ClipperPy35DevDockerfile (hunk 2 above)
Comment (withsmilo) [reply dockerfiles/ClipperPy35DevDockerfile:38]: This line is maybe needed.
```
 [py35-dev] Reading package lists... 
 [py35-dev] Building dependency tree... 
 [py35-dev] Reading state information... 
 [py35-dev] E: Unable to locate package openjdk-8-jre 
 [py35-dev] E: Unable to locate package openjdk-8-jdk-headless 
 [py35-dev] The command '/bin/sh -c mkdir -p /usr/share/man/man1 &&     apt-get install openjdk-8-jre openjdk-8-jdk-headless -y' returned a non-zero code: 100 
```

File: dockerfiles/ClipperDevDockerfile (hunk 3 above)
Comment (withsmilo) [reply dockerfiles/ClipperDevDockerfile:33]: This line is maybe needed.
```
 [dev] Reading package lists... 
 [dev] Building dependency tree... 
 [dev] Reading state information... 
 [dev] E: Unable to locate package openjdk-8-jre 
 [dev] E: Unable to locate package openjdk-8-jdk-headless 
 [dev] The command '/bin/sh -c mkdir -p /usr/share/man/man1 &&     apt-get install openjdk-8-jre openjdk-8-jdk-headless -y' returned a non-zero code: 100 
```

**Comment by RehanSD** [comment]
Lgtm

**Review by RehanSD — APPROVED** [review]

**Pull request merged**